nautobot/utilities/testing/views.py

```diff
@@ -799,6 +795,36 @@ def test_list_view_plugin_banner(self):
                 f"<div>You are viewing a table of {self.model._meta.verbose_name_plural}</div>", response_body
             )
 
+        @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
+        def test_queryset_to_csv(self):
+            # Built-in CSV export
+            if hasattr(self.model, "csv_headers"):
+                response = self.client.get(f"{self._get_url('list')}?export")
+                self.assertHttpStatus(response, 200)
+                self.assertEqual(response.get("Content-Type"), "text/csv")
+                instance1 = self._get_queryset().first()
+                # With filtering
+                response = self.client.get(f"{self._get_url('list')}?export&id={instance1.pk}")
+                self.assertHttpStatus(response, 200)
+                self.assertEqual(response.get("Content-Type"), "text/csv")
+                response_body = response.content.decode(response.charset)
+                reader = csv.DictReader(StringIO(response_body))
+                print(response_body)
+                data = [dict(row) for row in reader][0]
```

Comment:
This conversion is a bit unintuitive to me - maybe add a comment explaining what it's doing?